diff --git a/config/fcitx/config b/config/fcitx/config
index a7adff4..f0c6362 100644
--- a/config/fcitx/config
+++ b/config/fcitx/config
@@ -1,6 +1,6 @@
 [Hotkey]
 # 入力メソッドのオンオフ
-TriggerKey=CTRL_SPACE ZENKAKUHANKAKU
+TriggerKey=ZENKAKUHANKAKU
 # アクティブでない時だけその他のトリガーキーを使用する
 # 可能な値:
 # True False
@@ -31,13 +31,13 @@ TriggerKey=CTRL_SPACE ZENKAKUHANKAKU
 # カスタム切り替えキー
 #CustomSwitchKey=
 # 入力メソッドをオンに
-#ActivateKey=
+ActivateKey=HENKANMODE
 # 入力メソッドをオフに
-#InactivateKey=
+InactivateKey=MUHENKAN
 # 入力メソッドの切換をホットキーで
 # 可能な値:
 # True False
-#IMSwitchKey=True
+IMSwitchKey=False
 # 入力メソッドの切換にオフも含む
 # 可能な値:
 # True False
@@ -54,13 +54,13 @@ TriggerKey=CTRL_SPACE ZENKAKUHANKAKU
 # 2つのキー入力の間隔
 #TimeInterval=250
 # 仮想キーボードに切り替え
-#VKSwitchKey=CTRL_ALT_B
+VKSwitchKey=
 # リマインドモードの切り替え
 #RemindSwitchKey=
 # 全角文字モードに切り替え
 #FullWidthSwitchKey=
 # 全角句読点モードに切り替え
-#PuncSwitchKey=CTRL_.
+PuncSwitchKey=
 # 前のページ
 #PrevPageKey=- UP
 # 次のページ
@@ -68,9 +68,9 @@ TriggerKey=CTRL_SPACE ZENKAKUHANKAKU
 # 二番目と三番目の候補を選択
 #SecondThirdCandWordKey=
 # すべての設定と入力履歴を保存
-#SaveAllKey=CTRL_ALT_S
+SaveAllKey=
 # Switch Embedded Preedit
-#SwitchPreedit=CTRL_ALT_P
+SwitchPreedit=
 # 前の候補
 #PrevWord=SHIFT_TAB
 # 次の候補
